### PR TITLE
[DBIO-1652] fix: Lab account verification update status

### PR DESCRIPTION
**Pull request by bukandicki**
### JIRA Link
- https://blocksphere2020.atlassian.net/browse/DBIO-1652

### Changelog / Description
- Fix Lab account verification update status

### CODE REVIEW

File: src/views/Dashboard/Lab/index.vue [hunk 1]
```diff
@@ -39,16 +39,16 @@
...
             >mdi-check-decagram</v-icon>
             <v-icon 
               inline
               color="yellow darken-2"
               :size="20"
               v-else
             >mdi-information</v-icon>
-            <b v-if="labAccount.verificationStatus == 'Verified'">Your account has been verified</b>
-            <b v-else>Your account has not been verified</b>
+            <b v-if="labAccount && labAccount.verificationStatus == 'Unverified'">Your verification submission is being reviewed by Daogenic</b>
```
Comment (hilyds) [inline src/views/Dashboard/Lab/index.vue:50]: @bukandicki can you re-check the wording from Sarah / Haidar? If it's already correct, i just give a little input that the name should be DAOGenics, not Daogenic

Comment (bukandicki) [reply src/views/Dashboard/Lab/index.vue:50]: Hi @hilyds, according to Haidar, the wording is correct. Of course, I can change Daogenic to DAOGenics as soon as possible.
image

File: src/views/Dashboard/Lab/index.vue [hunk 2]
```diff
@@ -92,15 +92,23 @@ import LabOrders from "./LabOrders";
...
   computed: {
     ...mapState({
       labAccount: (state) => state.substrate.labAccount,
       isServicesExist: (state) => state.substrate.isServicesExist,
     }),
-  },
+
+    computeVerificationStatus() {
+      return this.labAccount?.verificationStatus
+        ? `Your account has been ${this.labAccount?.verificationStatus}`
```
Comment (agustinustheo) [inline src/views/Dashboard/Lab/index.vue:109]: Try this instead `Your lab account's verification status is: ${this.labAccount?.verificationStatus}`

**Review by agustinustheo — CHANGES_REQUESTED** [review]

File: src/views/Dashboard/Lab/index.vue (hunk 2 above)
Comment (bukandicki) [reply src/views/Dashboard/Lab/index.vue:109]: thanks, sure will update asap

**Review by agustinustheo — APPROVED** [review]

**Pull request merged**